app/assets/stylesheets/application.scss

```diff
@@ -13,26 +13,25 @@ $govuk-assets-path: "/";
 @import "admin";
 @import "bank-selection";
 @import "check-your-answers";
+@import "citizens/spinner";
```

Comment:
Thanks @agoldstone93 I would be careful reordering CSS imports alphabetically, this could cause styles to be overridden.

We should ideally implement a CSS architecture to help with this.